package.json

```diff
@@ -43,7 +43,8 @@
         "scss:clean": "grunt clean:scss",
         "serve:mock-service-for-android": "node ./src/tests/miscellaneous/mock-service-for-android/serve.js",
         "start:unified": "electron drop/electron/unified-dev/product/bundle/main.bundle.js",
-        "start:unified:dev": "cross-env DEV_MODE=true electron drop/electron/unified-dev/product/bundle/main.bundle.js",
+        "start:unified:dev": "cross-env DEV_MODE=true yarn start:unified",
+        "start:unified:mock-adb": "cross-env ANDROID_SDK_ROOT=drop/mock-adb yarn start:unified",
```

Comment:
Based on https://github.com/appium/appium-adb/blob/master/lib/helpers.js, ANDROID_HOME takes precedence over ANDROID_SDK_ROOT. To make sure that this will work for people who have ANDROID_HOME set in their environment, we should probably override ANDROID_HOME instead of ANDROID_SDK_ROOT